backend/authentication-service/app/config/db.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
```

Comment:
No issues for now, but once integrated there may be need for changes to the config